sigstore/_internal/tuf.py

```diff
@@ -207,9 +278,29 @@ def get_fulcio_certs(self) -> list[Certificate]:
 
         May download files from the remote repository.
         """
+        certs = []
+
+        self._trusted_root = self._get_trusted_root()
         # Return expired certificates too: they are expired now but may have
         # been active when the certificate was used to sign.
-        certs = self._get("Fulcio", ["Active", "Expired"])
+        if self._trusted_root:
+            for ca in self._trusted_root.certificate_authorities:
+                if not _is_timerange_valid(ca.valid_for, allow_expired=True):
+                    continue
```

Comment:
This is a no-op, right? If so maybe we can just omit it entirely, and perhaps just leave a comment to the effect of "we could check `_is_timerange_valid` here but don't bother to, since expired Fulcio chains are considered valid for verification purposes."

Reply:
Got it, thanks!